### PR TITLE
import: adjust for new local import

### PR DESCRIPTION
**Pull request by okJiang**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

- Added description for multiple and fewer columns, during local file import.
- Listed the options of the CSV configuration.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by Frank945946 — APPROVED** [review]

File: tidb-cloud/tidb-cloud-import-local-files.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ Currently, this method supports importing one CSV file for one task into either
 
     If the CSV file does not have a row for the column names, do not select **Use the first row as column name**. In this case:
 
-    - If the target table already exists, make sure the order of the columns is the same as the column list of the target table.
+    - If the target table already exists, the columns in the CSV file will be imported into the target table in order. Extra columns will be truncated and missing columns will be filled with default values. And you can also check the **Ignore the first row** option to ignore the first row and start importing from the second row.
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-import-local-files.md:49]:
```suggestion
    - If the target table already exists, the columns in the CSV file will be imported into the target table in order. Extra columns will be truncated and missing columns will be filled with default values. You can also select the **Ignore the first row** option to ignore the first row and start importing from the second row.
```

**Review by hfxsd — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

**Comment by qiancai** [comment]
@okJiang The Cloud doc sources are in the release-6.5 branch. Please rebase this PR to the release-6.5 branch.

**Comment by okJiang** [comment]
> @okJiang The Cloud doc sources are in the release-6.5 branch. Please rebase this PR to the release-6.5 branch.

Can we merge this pr first? Then github will create a pr automatically cherry-pick to release-6.5 branch

**Pull request merged**